python-scripts/update_hearings_with_caseReferenceNumber.py

```
@@ -0,0 +1,182 @@
+    "criteria": {"tenantId": "kl", "filingNumber": "KL-000759-2025"},
+    "RequestInfo": {
+        "apiId": "Rainmaker",
+        "authToken": auth_token,
+        "msgId": "1730970507148|en_IN",
+        "plainAccessRequest": {},
+    },
+}
+try:
+    hearing_response = requests.post(
+        env_url + hearing_search_endpoint, headers=headers, json=hearing_search_request
+    ).json()
```

Pylint (3.3.7)

[warning] Missing timeout argument for method 'requests.post' can cause your program to hang indefinitely

(W3101)